backend/tests/routes.test.js

```diff
@@ -0,0 +1,125 @@
+// FILE routes.test.js
+
+/**
+ * @jest-environment node
+ */
+const axios = require('axios');
+const mongoose = require('mongoose');
+const usersModel = require('../models/users');
+
+beforeAll(async () => {
+  const dbuser = process.env.dbuser;
```

Comment:
How are you getting environment variables without the dotenv package? I tried running the backend and these tests on my machine and found that the package isn't being imported.

Reply:
Same here.